homeassistant/components/flunearyou/diagnostics.py

```diff
@@ -0,0 +1,40 @@
+"""Diagnostics support for Flu Near You."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import CATEGORY_CDC_REPORT, CATEGORY_USER_REPORT, DOMAIN
+
+TO_REDACT = {
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    coordinators: dict[str, DataUpdateCoordinator] = hass.data[DOMAIN][entry.entry_id]
+
+    return {
+        "entry": {
+            "title": entry.title,
+            "data": async_redact_data(entry.data, TO_REDACT),
+        },
```

Comment:
You could simple add the whole config entry (_check if further data needs to be redacted_)

```suggestion
        "entry": "async_redact_data(config_entry.as_dict(), REDACT_CONFIG),
```